documentation/configuration.markdown

```diff
@@ -121,55 +123,67 @@ To change the JRuby version Puppet Server uses, you can edit the init config fil
 
 ### Enabling JRuby 9k
 
-1. Open the init config file and add the following line:
+1.  Open the init config file and add the following line:
```

Comment:
What is the "init config file"? If we don't explain that term elsewhere then users will probably find that confusing.